openspec/specs/sessions/spec.md

```
@@ -382,43 +562,34 @@ Then the response status MUST be 401
-```
+- GIVEN the runtime is backed by a non-SQLite backend
+- WHEN slash-session persistence operations are requested
+- THEN the runtime MUST return an explicit unsupported result
+- AND the operation MUST NOT report success.
 
 ## Change History
 
 | Version | Date       | Changes                                                     |
 |---------|------------|-------------------------------------------------------------|
-| 1.0.0   | 2026-03-28 | Initial specification from session-memory-visibility change |
+| 1.0.0   | 2026-03-28 | Initial specification from session-memory-visibility change |
```

markdownlint-cli2 (0.22.0)

[warning] Files should end with a single newline character

(MD047, single-trailing-newline)